src/Pos/Binary/Relay.hs

```diff
@@ -47,3 +62,15 @@ instance Bi (DataMsg (UpdateProposal, [UpdateVote])) where
 instance Bi (DataMsg UpdateVote) where
     put (DataMsg uv) = put uv
     get = label "DataMsg UpdateVote" $ DataMsg <$> get
+
+instance Bi (DataMsg ProxySKLight) where
+    put (DataMsg p) = put p
+    get = label "DataMsg ProxySKLight" $ DataMsg <$> get
+
+instance Bi (DataMsg ProxySKHeavy) where
+    put (DataMsg p) = put p
+    get = label "DataMsg ProxySKHeavy" $ DataMsg <$> get
+
+instance Bi (DataMsg (ProxySKLight, ProxySigLight ProxySKLight)) where
```

Comment:
It would be good to wrap `(ProxySKLight, ProxySigLight ProxySKLight)` into something meaningful. At least add a type alias.